tests/e2e/set-ffi-env-e2e

```diff
@@ -190,21 +190,24 @@ setup_qm_services() {
   cat > /etc/bluechi/controller.conf << 'EOF'
 [bluechi-controller]
 AllowedNodeNames=qm.localrootfs,localrootfs
-ControllerPort=842
+ControllerPort=8420
 LogLevel=INFO
 LogTarget=journald
 LogIsQuiet=false
 EOF
 cat > /etc/bluechi/agent.conf.d/00-default.conf << 'EOF'
 [bluechi-agent]
 NodeName=localrootfs
+ControllerPort=8420
 EOF
 
+# Insert ControllerHost and ControllerPort into /etc/qm/bluechi/agent.conf.d/agent.conf
 controller_host_ip=$(hostname -I | awk '{print $1}')
 qm_bluechi_agent_config_file="/etc/qm/bluechi/agent.conf.d/agent.conf"
 if [[ -f "${qm_bluechi_agent_config_file}" ]]; then
     if ! grep "ControllerHost=${controller_host_ip}" "${qm_bluechi_agent_config_file}" >/dev/null; then
         sed -i '$a \ControllerHost='"${controller_host_ip}"'' ${qm_bluechi_agent_config_file}
+        sed -i '$a \ControllerPort=8420' ${qm_bluechi_agent_config_file}
```

Comment:
Do you think it is better to move `ControllerPort=8420` to `bluechiSetup()` of `setup`